chia/wallet/cat_wallet/cat_utils.py

```diff
@@ -41,11 +41,15 @@ def match_cat_puzzle(puzzle: Program) -> Tuple[bool, Iterator[Program]]:
         return False, iter(())
 
 
-def construct_cat_puzzle(mod_code: Program, limitations_program_hash: bytes32, inner_puzzle: Program) -> Program:
+def construct_cat_puzzle(
```

Comment:
aren't there any call sites to this function that needs updating too?

Reply:
yes but they are not so performance critical so i chose to make the patch smallerinstead